install_config/aggregate_logging.adoc

```diff
@@ -750,32 +747,23 @@ As part of the OpenShift cluster installation, it is recommended that you add th
 Fluentd node selector to the list of persisted
 xref:../install_config/install/advanced_install.adoc#configuring-node-host-labels[node labels].
 
-[[fluentd-use-journald]]
-*Having Fluentd Use the Systemd Journal as the Log Source*
+Fluentd uses `journald` as the system log source. These are log
+messages from the operating system, from Docker, and from OpenShift. For
```

Comment:
s/from the operating system, from Docker, and from OpenShift/from the operating system, Docker, and OpenShift ??